e2e/basic/suite_setup_test.go

```diff
@@ -21,31 +17,14 @@ import (
 const (
 	testTimeout = time.Minute * 15 // the same time that is used in the ci/cd pipeline
 
-	nginxImage       = "docker.io/nginx:latest"
-	nginxVolumeOwner = 0
-	nginxPort        = 80
-	nginxHTMLPath    = "/usr/share/nginx/html"
-	nginxCommand     = "nginx -g daemon off"
-
 	resourcesHTML = "../system/resources/html"
-
-	alpineImage = "alpine:latest"
+	alpineImage   = "alpine:latest"
 )
 
 type Suite struct {
```

Comment:
Why do we still need this here?

Reply:
Is this to parallelize the tests?

Reply:
If you mean the `Suite struct` here, yes we need it to have separate test suite per test category that we have and the common parts are extracted to the suite pkg in the e2e.